fusestore/store/src/meta_service/meta.rs

```diff
@@ -236,6 +277,11 @@ impl Meta {
         let x = self.databases.get(name);
         x.cloned()
     }
+
+    pub fn get_unclassified(&self, name: &str) -> Option<(u64, String)> {
```

Comment:
replace `&str` with `&[u8]`?

Reply:
For value `Vec<u8>` is better, since it could any binary values.
For key, it should be something human readable and unlikely it is an invalid utf8 byte series.
So, what about leaving the key a `String`?

Reply:
IMHO, even for key,  `Vec<u8>` is better than `String`, as we are providing a *generic* KV API. 
If the caller (of API) can be sure of the internal encoding of key, take utf-8 string for example, the caller can always convert the bytes to utf8 String by themself. 
This may bring some inconveniences to Store Server, like logging messages I think, but that's acceptable for me, that's the affordable cost for being generic :)

Reply:
sorry for my late reply, found that you have already pushed a commit

Reply:
A readable log is quite important in a large scale system.
Unreadable log worth nothing.
Generalization is mathematically beautiful but without explicit need it is too high a price to pay but gained nothing.

If there are cases in which byte slice key is needed, uri-encoding is an option too.

Reply:
Yes, I totally agree with you that logging is important.

But utf8 encoding (as rust `String` `str` implies) is too restrictive IMHO, (e.g. API user may want to concat a prefix with a uuid or hash as key, and the uuid/hash is encoded as bytes, not hex characters, to save some space,  or just do not want another layer of en/decode logic involved...). 

anyway, we can always refactor it later if needed.

let's move :D

Reply:
Right. 
Once there emerges explicit need for u8 slice key or there are cases in which encoded key length is critical, the refactoring won't be difficult(simple as flipping a hand? 🤔 )